BuildingSync.xsd

```diff
@@ -14777,6 +14803,21 @@ Window spacing: the dimension between windows in a discrete window layout. (in.)
 			</xs:restriction>
 		</xs:simpleType>
 	</xs:element>
+	<xs:element name="BurnerControlType">
+		<xs:annotation>
+			<xs:documentation>Control type of burner, if applicable.</xs:documentation>
+		</xs:annotation>
+		<xs:simpleType>
+			<xs:restriction base="xs:string">
+				<xs:enumeration value="Full Modulation Set to Manual"/>
```

Comment:
@supriyagoel Is this burner control type what we really want here? We're not too sure about this name.